staging_docs/admin/guides/remove-content.md

```diff
@@ -0,0 +1,26 @@
+# Remove Content from Pulp
+
+Removing content from Pulp entirely is slightly complicated for a number of architectural reasons.
+
+1. *Repository Versions* are immutable, so you can get back to any version and have all the content available. This means that it is not possible to remove content as long as there still exists a repo-version that "knows about it".
+2. Pulp deduplicates content. Artifacts can be shared across repositories - which means the repo being looked at, may not be the only one whose repo-versions still point to the content.
+
+The way content gets removed "from Pulp" is via the orphan cleanup task, which will look for Artifacts that don't belong to any repository-versions, and haven't been touched in a while, and removes them.
+
+!!! warning
+
+    Please, follow the script steps carefully and **only if you know what you are** doing because you risk losing data.
+
+```bash
+#!/usr/bin/env bash
+
+# Remove content from repository
+echo "Removing content from ${REPO_NAME}"
+pulp rpm repository version destroy --repository $REPO_NAME
+
+echo "Removing orphan rpm contents"
+pulp orphan cleanup --content-hrefs $(pulp rpm content list | jq -cr '.|map(.pulp_href)') --protection-time 0
```

Comment:
@pedro-psb @mdellweg So do we want to leave this, as-is, or exist only in the pulpcore docs?  Should we note that you generally shouldn't use `--orphan-protection-time 0`?

Reply:
Good point. There is really nothing specific to rpm here. And we can reference pulpcore docs from here (hasn't this been the whole point of the docs transition?).

I'm conflicted about the protection time. Using `--content-hrefs` this is already not "general" anymore. But still you can break an ongoing sync that way (much less likely though). My personal preference of the whole matter (and i hear echoes that we can never do this) is don't talk about it at all, teach people that content only lives _in_ repositories, automatically schedule orphan cleanup, rewrite sync so that no orphans are created on the way and remove the orphan cleanup api altogether. The fact that we constantly come back discussing how weird and dangerous this is keeps reinforcing me. Having said all that, I am unable to express a useful opinion on how to  document this thing that should never have been there in the first place.

Reply:
My personal feeling is that we should probably drop the orphan removal aspect.  We could link to it, though

Reply:
So it looks like we don't really cover orphan cleanup in the pulpcore docs.  IMO, we probably should, and then perhaps link it from the plugin docs when relevant?

The original justification was likely: the workflow for every plugin is largely different (how sync works, how publish works and whether it's necessary, etc.) so most of the actual use case docs are in plugins.  Orphan cleanup is pretty generic though, so it feels to me like it belongs in pulpcore.